cmd/kubeadm/app/constants/constants.go

```diff
@@ -555,3 +552,36 @@ func GetDNSVersion(dnsType kubeadmapi.DNSAddOnType) string {
 func GetKubeletConfigMapName(k8sVersion *version.Version) string {
 	return fmt.Sprintf("%s%d.%d", KubeletBaseConfigurationConfigMapPrefix, k8sVersion.Major(), k8sVersion.Minor())
 }
+
+// GetCurrentKubernetesVersion returns current Kubernetes version supported by kubeadm
+func GetCurrentKubernetesVersion() *version.Version {
+	gitVersion := pkgversion.Get().GitVersion
+	if strings.Contains(gitVersion, "-") {
+		gitVersion = strings.Split(gitVersion, "-")[0]
+	}
+	// If bazel or hack/lib/version.sh does not generate the correct version number,
+	// use hardcoded CurrentKubernetesVersion
+	// TODO: Get rid of hardcoded version
```

Comment:
nit. I found this comment confusing, and in general don't like `//TODO:`'s in the code with a preference to opening a ticket if it's not going to be done now.